ci/build-unix.yml

```diff
@@ -5,6 +5,7 @@ parameters:
   is_release: ''
   name: ''
   release_tag: ''
+  scala_version: '2.12.12'
```

Comment:
I'd love to have this hard-coded in fewer places. Can we set it to the empty string by default? We might need to handle this in _scala_version.bzl_.

Reply:
Very valid point 9ea71dda2e5e7e59becd037149b96736f2f4bf9b

Reply:
Yay!